tutoraspects/templates/aspects/apps/superset/pythonpath/create_row_level_security.py

```diff
@@ -1,15 +1,8 @@
-#!/usr/bin/env bash
-set -e
+from superset.app import create_app
```

Comment:
Hi @Ian2012 -- this is looking really beautiful :)
But when I launched a new `tutor local` from scratch, I got an error during init:

```
Traceback (most recent call last):
  File "/app/pythonpath/create_row_level_security.py", line 39, in <module>
    assert table, f"{schema}.{table_name} table doesn't exist yet?"
AssertionError: xapi.xapi_events_all_parsed table doesn't exist yet?
```

Reply:
Thanks for the review, it was due to the order of the init scripts. (superset before clickhouse). I just move the clickhouse job before the superset job :)

Reply:
Perfect! That did the trick :)